package.json

```diff
@@ -10,7 +10,7 @@
   },
   "devDependencies": {
     "@types/jest": "^24.0.25",
-    "@types/node": "^12.12.17",
+    "@types/node": "^13.1.4",
```

Comment:
Hmmm, This one I think should stay back at 12 as we're using 12?

Reply:
@tteltrab Thoughts on this one?

Reply:
Yeah, I think we want to pin this one then? or at least tell greenkeeeper to ignore it, but so long as we're using nvm and our node version is pinned it makes sense for this to be the same version I think?